src/scenes/preloader-scene.ts

```diff
@@ -0,0 +1,74 @@
+// import Phaser from 'phaser';
```

Comment:
Do we not need this import in this class now? Can delete if not needed anymore

Reply:
yeah i'm not sure exactly why, but it seems to me that the Phaser imports are no longer needed when the are only needed for TS types in the file...

as TS is only used in IDE and at compile time i guess it's a bit more smart, and can auto import / comb all the types somehow - so thats why it's removed, perhaps we could remove many of these